docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -755,24 +755,208 @@ To read and write a value of the Bluetooth descriptor:
 ### Accessing connection's ATT MTU
 <!-- END #11 -->
 
-<!-- TODO #12
-      - write about registerService()
- -->
 ### Registering services
-<!-- END #12 -->
+To register a GATT service in the local server:
+
+1. Define the init data of the service and all of its components:
+
+   ```
+   var descriptorInitData = {
+     uuid: 'abcd'
+     /* Define more attributes, if needed */
+   };
+
+   var characteristicInitData = {
+     uuid: '9876',
+     descriptors: [descriptorInitData]
+     /* Define more attributes, if needed */
+   };
+
+   var includedServiceInitData = {
+     serviceUuid: '5678',
+     isPrimary: true
+   };
+
+   var serviceInitData = {
+     serviceUuid: '1234',
+     includedServices: [includedServiceInitData],
+     characteristics: [characteristicInitData],
+     isPrimary: true
+   };
+   ```
+
+2. Register the service in the local server:
+
+   ```
+   function successCallback() {
+     console.log('Service registered successfully');
+   }
+
+   function errorCallback(error) {
+     console.log('Registering GATT service failed: ' + error.message);
+   }
+
+   var server = tizen.bluetooth.getGATTServer();
+   server.registerService(serviceInitData, successCallback, errorCallback);
+   ```
+   > **Note**
+   > The service will be registered in the server until one of the following
+   > happens:
+   > - service is unregistered with its `unregister()` method,
+   > - all services are unregistered with server's `unregisterAllServices()` method,
+   > - application is reloaded,
+   > - application is closed.
```

Comment:
I think that if app is closed you can't execute any code from that app.

```suggestion
   > - application was closed.
```

Reply:
1. Yes, the user can't execute any code. The service will be unregistered automatically, when app is closed - i.e. clients won't be able to access it.

2. Why should we change "is" to "was"? Currently it's a sentence in the first conditional https://learnenglish.britishcouncil.org/grammar/intermediate-to-upper-intermediate/conditionals-1